spec/lib/sms_api_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require "savon/mock/spec_helper"
+
+describe SMSApi do
+  let(:api) { SMSApi.new }
+
+  context "#sms_deliver" do
+    include Savon::SpecHelper
+
+    before do
+      endpoint = "https://intranet.lorca.es/wssms/wssms.asmx?WSDL"
```

Comment:
We don't connect to this domain in the test, right? To avoid confusion, how about using `example.com` or `localhost`?

Reply:
Savon connects to the WSDL endpoint to fetch the web service information even when mocking the operation response. 

It seems there is an [open issue](https://github.com/savonrb/savon/issues/396) regarding this topic. Maybe we can mock the WSDL response by mocking it as this [comment](https://github.com/savonrb/savon/issues/396#issuecomment-142959178) suggests.